main.go

```diff
@@ -1,11 +1,29 @@
 package main
 
 import (
+	"os"
+	"os/signal"
+	"syscall"
+
 	logger "github.com/kubescape/go-logger"
 	"github.com/kubescape/kubescape/v3/cmd"
 )
 
 func main() {
+	// Capture interrupt signal
+	signalChan := make(chan os.Signal, 1)
+	signal.Notify(signalChan, os.Interrupt, syscall.SIGTERM)
+
+	// Handle interrupt signal
+	go func() {
+		<-signalChan
+		// Perform cleanup or graceful shutdown here
+		logger.L().StopError("Received interrupt signal, exiting...")
+
+		// Exit the program
+		os.Exit(0)
```

Comment:
shouldn't it be exit 1 for interrupt?

Reply:
Oops, my bad. Thanks for pointing this out.

While we're at it, do you think we should use exit code 130 (As it is reserved for interrupt signals)?
https://itsfoss.com/linux-exit-codes/

Reply:
![image](https://github.com/kubescape/kubescape/assets/34866732/f927bdd1-8e9f-4bf4-9da7-a5425906ca21)

Reply:
ok!

Reply:
I've updated the code. On a side note, I tried to write tests for this change but since the program exits after receiving an interrupt signal, I'm having a hard time figuring out the right way to refactor the code and write the tests.

Reply:
don't worry I'll merge like that